app/uk/gov/hmrc/cdsreimbursementclaimfrontend/views/fileupload/choose_file.scala.html

```diff
@@ -0,0 +1,67 @@
+@*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import play.api.i18n.Messages
+@import play.twirl.api.Html
+@import play.api.mvc.Call
+@import play.api.mvc.Request    
+@import uk.gov.hmrc.cdsreimbursementclaimfrontend.config.ViewConfig
+@import uk.gov.hmrc.cdsreimbursementclaimfrontend.models.upscan.UpscanUpload
+@import uk.gov.hmrc.cdsreimbursementclaimfrontend.models.UploadDocumentsSessionConfig
+@import uk.gov.hmrc.cdsreimbursementclaimfrontend.views.utils.MessagesHelper._
+@import uk.gov.hmrc.cdsreimbursementclaimfrontend.views.components.hints.DropdownHints
+
+@this(
+    layout: uk.gov.hmrc.cdsreimbursementclaimfrontend.views.html.Layout,
+    pageHeading: uk.gov.hmrc.cdsreimbursementclaimfrontend.views.html.components.page_heading,
+    paragraph: uk.gov.hmrc.cdsreimbursementclaimfrontend.views.html.components.paragraph_block,
+    inputFile: uk.gov.hmrc.cdsreimbursementclaimfrontend.views.html.components.input_file,
+    submitButton: uk.gov.hmrc.cdsreimbursementclaimfrontend.views.html.components.button,
+    dropdownDetails : uk.gov.hmrc.cdsreimbursementclaimfrontend.views.html.components.dropdown_details
+)
+
+@(upscanUpload : UpscanUpload, sessionConfig: UploadDocumentsSessionConfig)(implicit request : Request[_], messages: Messages, viewConfig: ViewConfig)
+
+    @meta = @{upscanUpload.upscanUploadMeta.uploadRequest.fields}
+    @href = @{upscanUpload.upscanUploadMeta.uploadRequest.href}
+
+    @layout(pageTitle = Some(sessionConfig.content.title)) {
+
+        @pageHeading(sessionConfig.content.title)
```

Comment:
Shouldn't the text for the title be in the messages file so we can also display it in welsh?

Reply:
the title is indeed in the messages and then passed as part of an upload session config